ReactNative/ReactTwitter/android/app/build.gradle

```diff
@@ -121,6 +122,8 @@ android {
 
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
+    compile project(':react-native-push-notification')
+    compile 'com.google.android.gms:play-services-gcm:8.3.0'
```

Comment:
'9.0.0' is available.
The current one is enough for our proof of concept, but do you want to give it a try?